recipes/sd/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "0.7.6" %}
+
+package:
+  name: sd
+  version: {{ version }}
+
+source:
+  url: https://github.com/chmln/sd/archive/v{{ version }}.tar.gz
+  sha256: faf33a97797b95097c08ebb7c2451ac9835907254d89863b10ab5e0813b5fe5f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
```

Comment:
Why do you need this?

Reply:
Not necessarily, but checking the `meta.yml` file in three other packaged rust projects they all have it. I could try removing it and see if the automated tests don't fail.

Reply:
This is now changed to a copy-pasta from `bat`